plonky2/src/fri/batch_oracle.rs

```diff
@@ -83,6 +85,7 @@ impl<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize>
                     PolynomialBatch::<F, C, D>::lde_values(
                         &polynomials[group_start..i + 1],
                         rate_bits,
+                        shift.exp_power_of_2(max_degree_log - d),
```

Comment:
Adjust the shift so that the evaluation domain matches with corresponding round in FRI reduction step.